helpers/createChain.sh

```diff
@@ -0,0 +1,76 @@
+#! /bin/bash
+
+source lib/utils/helpers/common.sh
+
+function usage {
+  echo "bash createChain.sh <chain> <?governor> <?guardian>"
+  echo ""
+  echo -e "chain: chain to deploy on"
+  echo -e "\t0: Fork"
+  echo -e "\t1: Ethereum Mainnet"
+  echo -e "\t2: Arbitrum"
+  echo -e "\t3: Polygon"
+  echo -e "\t4: Gnosis"
+  echo -e "\t5: Avalanche"
+  echo -e "\t6: Base"
+  echo -e "\t7: Binance Smart Chain"
+  echo -e "\t8: Celo"
+  echo -e "\t9: Polygon ZkEvm"
+  echo -e "\t10: Optimism"
+  echo -e "\t11: Linea"
+  echo -e "governor: address of the governor (optional)"
+  echo -e "guardian: address of the guardian (optional)"
+  echo ""
+}
+
+function main {
+    if [ $# -ne 3 ] && [ $# -ne 5 ]; then
```

Comment:
I don't understand what this does?

Reply:
do you mean the if or the whole function ?

Reply:
in the case of the if it checks if it has the right number of arguments